keyboards/m65/rev2/mcuconf.h

```diff
@@ -0,0 +1 @@
+../rev1/mcuconf.h
```

Comment:
Please delete this file, and the readme.md.

Reply:
done thank you, removed symlink added the file